app/src/main/java/com/sun/noteapp/ui/trash/TrashActivity.kt

```diff
@@ -0,0 +1,208 @@
+                    createRestoreDialog(selectedItems)
+                }
+                R.id.option_item_select_all -> {
+                    selectedItems.clear()
+                    for (position in 0 until trashAdapter.itemCount)
+                        selectedItems.add(position)
+                    trashAdapter.selectAll()
+                }
+            }
+            false
+        }
+        toolbarSelector.visibility = View.GONE
+    }
+
+    fun closeToolBarSelector() {
+        selectedItems.clear()
+        supportActionBar?.show()
+        toolbarSelector.visibility = View.GONE
+    }
+
+    private fun changeToolbar() {
+        if (selectedItems.isEmpty()) {
+            supportActionBar?.show()
+            toolbarSelector.visibility = View.GONE
+        } else {
+            supportActionBar?.hide()
+            toolbarSelector.visibility = View.VISIBLE
+        }
+    }
+
+    private fun enableSwipeToDelete() {
+        val swipeItemCallback = object : SwipeItemCallback(this) {
+            override fun onSwiped(viewHolder: RecyclerView.ViewHolder, direction: Int) {
+                val position = viewHolder.adapterPosition
+                if (direction == SWIPE_RTL) {
+                    //presenter.deleteNotes(arrayListOf(position))
+                    createDeleteDialog(arrayListOf(position))
+                } else {
+                    //presenter.restoreNotes(arrayListOf(position))
+                    createRestoreDialog(arrayListOf(position))
+                }
+            }
+        }
+        val itemTouchHelper = ItemTouchHelper(swipeItemCallback)
+        itemTouchHelper.attachToRecyclerView(recyclerTrash)
+    }
+
+    private fun showSortDialog() {
+        SortDialog(this, R.layout.dialog_sort, object : BaseDialog.OnLoadDialogCallback<String> {
+            override fun onSuccess(parrams: String) {
+                sortType = parrams
+                presenter.getAllHideNote(sortType)
+            }
+        }).show()
+    }
+
+    private fun createDeleteDialog(listPosition: List<Int>) {
+        val deleteDialog = AlertDialog.Builder(this)
+        deleteDialog.setMessage("Do you want to delete note?")
+        deleteDialog.setPositiveButton("yes", object : DialogInterface.OnClickListener {
+            override fun onClick(p0: DialogInterface?, p1: Int) {
+                presenter.deleteNotes(listPosition)
+                closeToolBarSelector()
+            }
+        })
+        deleteDialog.setNegativeButton("no", object : DialogInterface.OnClickListener {
+            override fun onClick(p0: DialogInterface?, p1: Int) {
+                trashAdapter.removeAllSelection()
+                closeToolBarSelector()
+            }
+        })
+        deleteDialog.show()
+    }
+
+    private fun createRestoreDialog(listPosition: List<Int>) {
+        val restoreDialog = AlertDialog.Builder(this)
+        restoreDialog.setMessage("Do you want to restore note?")
```

Comment:
Builder thì viết liền vào thôi chứ.
```suggestion
        AlertDialog.Builder(this)
            .setMessage("Do you want to restore note?")
            .setXY()
            .show()
```